pdcs/containers/stats.go

```diff
@@ -0,0 +1,22 @@
+package containers
+
+import (
+	"github.com/containers/podman-tui/pdcs/connection"
+	"github.com/containers/podman/v3/pkg/bindings/containers"
```

Comment:
A mere drive-by-comment: at some point, we may need to bump the imported Podman to `v4` so the tui can talk to Podman 4.0+

Reply:
You are right.
I was thinking to add support for podman machine and remote features and then go to v4, but maybe better to bump it to v4 first since podman 4.0 is out.